modules/http/src/main/java/org/mule/extension/http/api/policy/HttpRequestPolicyPointcutParameters.java

```diff
@@ -46,4 +50,11 @@ public String getPath() {
   public String getMethod() {
     return method;
   }
+
+  /**
+   * @return the name of the flow executing the http request
+   */
+  public String getFlowName() {
+    return flowName;
```

Comment:
We are using the flow name in our AutoDiscovery element to map the API in Platform with the implementation. So we need the flow name to compare against the one in the AutoDiscovery element when a policy has an API Pointcut. 
We have in mind to re-think that element, so not sure if we are going to need it after that, but we need it for the next release at least